src/redux/saga/index.ts

```diff
@@ -0,0 +1,18 @@
+import { takeEvery } from 'redux-saga/effects';
+import { AuthSagaActions } from '../actions/authSagaActions';
+import { signInSaga } from './signInSaga';
+import { signOutSaga } from './signOutSaga';
+import { TodoSagaActions } from '../actions/todoSagaActions';
+import { addItemSaga } from './adItemSaga';
+import { doneItemSaga } from './doneItemSaga';
+import { deleteItemSaga } from './deleteItemSaga';
+import { getDataTodoItemsSaga } from './getDataTodoItemsSaga';
+
+export function* rootWatcher() {
+  yield takeEvery(AuthSagaActions.SIGN_IN_SAGA, signInSaga);
```

Comment:
please check the differents between takeEvery and takeLatest and think what  effect  better

Reply:
ok